[2025-10-07T00:41:28.427+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-07T00:41:28.433+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Titanic_ML_DAG.evaluate_model manual__2025-10-07T00:41:22.878133+00:00 [queued]>
[2025-10-07T00:41:28.436+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Titanic_ML_DAG.evaluate_model manual__2025-10-07T00:41:22.878133+00:00 [queued]>
[2025-10-07T00:41:28.436+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-10-07T00:41:28.441+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): evaluate_model> on 2025-10-07 00:41:22.878133+00:00
[2025-10-07T00:41:28.446+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=244) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-07T00:41:28.445+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'Titanic_ML_DAG', 'evaluate_model', 'manual__2025-10-07T00:41:22.878133+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/titanic_dag.py', '--cfg-path', '/tmp/tmp8wwwyofc']
[2025-10-07T00:41:28.447+0000] {standard_task_runner.py:105} INFO - Job 31: Subtask evaluate_model
[2025-10-07T00:41:28.446+0000] {standard_task_runner.py:72} INFO - Started process 255 to run task
[2025-10-07T00:41:28.464+0000] {task_command.py:467} INFO - Running <TaskInstance: Titanic_ML_DAG.evaluate_model manual__2025-10-07T00:41:22.878133+00:00 [running]> on host 8587a8a43883
[2025-10-07T00:41:28.586+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mihir' AIRFLOW_CTX_DAG_ID='Titanic_ML_DAG' AIRFLOW_CTX_TASK_ID='evaluate_model' AIRFLOW_CTX_EXECUTION_DATE='2025-10-07T00:41:22.878133+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-07T00:41:22.878133+00:00'
[2025-10-07T00:41:28.586+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-07T00:41:28.594+0000] {logging_mixin.py:190} INFO - âœ… Model accuracy on full data: 0.80
[2025-10-07T00:41:28.594+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-07T00:41:28.597+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-07T00:41:28.597+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=Titanic_ML_DAG, task_id=evaluate_model, run_id=manual__2025-10-07T00:41:22.878133+00:00, execution_date=20251007T004122, start_date=20251007T004128, end_date=20251007T004128
[2025-10-07T00:41:28.637+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-07T00:41:28.644+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-07T00:41:28.644+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
